app/views/admin/events/show.html.haml

```diff
@@ -71,7 +71,7 @@
             %td
               - if @event.speakers.count == 1
                 = link_to 'Fill out speaker\'s biography', edit_admin_user_path(@event.speakers.first)
-              - else
+              - elsif @event.speakers.count > 1
```

Comment:
[`count`](https://apidock.com/rails/ActiveRecord/Associations/CollectionProxy/count) raises a query every time it is called. So you shouldn't call it twice. You can use the [case statement](https://www.rubyguides.com/2015/10/ruby-case/) for this. :wink: